data/xml/2014.amta.xml

```diff
@@ -0,0 +1,677 @@
+      <title>Term translation central: up-to-date <fixed-case>MT</fixed-case> without frequent retraining</title>
+      <author><first>Ventsislav</first><last>Zhechev</last></author>
+      <pages>152-159</pages>
+      <attachment type="presentation" hash="c0f8cea9">2014.amta-users.11.Presentation.pdf</attachment>
+    </paper>
+    <paper id="12">
+      <title>Translation technology in action: a <fixed-case>US</fixed-case> government use case</title>
+      <author><first>Vanesa</first><last>Jurica</last></author>
+      <pages>160-180</pages>
+      <attachment type="presentation" hash="407355e2">2014.amta-users.12.Presentation.pdf</attachment>
+    </paper>
+    <paper id="13">
+      <title>Machine translation for e-government – the Baltic case</title>
+      <author><first>Andrejs</first><last>Vasiļjevs</last></author>
+      <author><first>Rihards</first><last>Kalniņš</last></author>
+      <author><first>Mārcis</first><last>Pinnis</last></author>
+      <author><first>Raivis</first><last>Skadiņš</last></author>
+      <pages>181-193</pages>
+      <attachment type="presentation" hash="07d1f5c6">2014.amta-users.13.Presentation.pdf</attachment>
+    </paper>
+    <paper id="14">
+      <title>Panel: Inserting <fixed-case>CAT</fixed-case> tools into a government <fixed-case>LSP</fixed-case> environment</title>
+      <author><first>Tanya</first><last>Helmen</last></author>
+      <author><first>Vanesa</first><last>Jurica</last></author>
+      <author><first>Danielle</first><last>Silverman</last></author>
+      <author><first>Elizabeth</first><last>Richerson</last></author>
+      <pages>194-202</pages>
+      <attachment type="presentation" hash="0b3770b0">2014.amta-users.14.Presentation.pdf</attachment>
+    </paper>
+    <paper id="15">
+      <title>A novel use of <fixed-case>MT</fixed-case> in the development of a text level analytic for language learning</title>
+      <author><first>Carol</first><last>Van Ess-Dykema</last></author>
+      <author><first>Salim</first><last>Roukos</last></author>
+      <author><first>Amy</first><last>Weinberg</last></author>
+      <pages>203-212</pages>
+      <attachment type="presentation" hash="92af34aa">2014.amta-users.15.Presentation.pdf</attachment>
+    </paper>
+    <paper id="16">
+      <title>Technology showcase guide</title>
+      <author><first>Jennifer</first><last>DeCamppp</last></author>
+      <attachment type="presentation" hash="7bf083ea">2014.amta-users.16.Presentation.pdf</attachment>
+    </paper>
+  </volume>
+  <volume id="tutorials" ingest-date="2021-06-20">
+    <meta>
+      <booktitle>Proceedings of the 11th Conference of the Association for Machine Translation in the Americas: Tutorials</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Vancouver, Canada</address>
+      <month>October 22-26</month>
+      <year>2014</year>
+      <editor><first>Yaser</first><last>Al-Onaizan</last></editor>
```

Comment:
Not sure who to put here. Where did you get this list?

Reply:
I got the long list of editors from the google sheet. This seems like a mistake since it is not relevant for different parts of the conference. I'll remove it.